src/rust/plugin-work-queue/src/test_utils.rs

```diff
@@ -99,62 +99,66 @@ impl PsqlQueueTestExtensions for PsqlQueue {
 pub async fn scan_for_generator_plugin_message_in_pwq(
     psql_queue: PsqlQueue,
     plugin_id: uuid::Uuid,
-) -> Option<NextExecutionRequest> {
```

Comment:
as it turns out, this as an Option never made sense - it was either Some or panic!
so now it's Ok or Err